addons/stock/wizard/stock_quant_relocate.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-is_not_full_package
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+class RelocateStockQuant(models.TransientModel):
+    _name = 'stock.quant.relocate'
+    _description = 'Stock Quantity Relocation'
+
+    quant_ids = fields.Many2many('stock.quant')
+    company_id = fields.Many2one(related="quant_ids.company_id")
+    location_id = fields.Many2one('stock.location', domain="[('usage', '=', 'internal'), ('company_id', '=', company_id)]")
+    allowed_package_ids = fields.Many2many('stock.quant.package', compute="_compute_allowed_package_ids")
+    package_id = fields.Many2one('stock.quant.package', domain="[('id', 'in', allowed_package_ids)]")
```

Comment:
That kind of domain is always problematic with the performance. We should try to have a good compromise with a static domain

Reply:
done